docs/dm/zh_CN/configuration/configuration.md

```diff
@@ -64,14 +67,23 @@ filters:                                            # 上游数据库实例的
     ​    table-pattern: "t_*"
     ​    events: ["truncate table", "drop table"]
     ​    action: Ignore
+    user-filter-2:
```

Comment:
s/user-filter-2/filter-rule-2
it is better to keep same name regulation with `filter-rule-1`